components/style/domrefcell.rs

```diff
@@ -11,12 +11,19 @@ use thread_state;
 ///
 /// This extends the API of `core::cell::RefCell` to allow unsafe access in
 /// certain situations, with dynamic checking in debug builds.
-#[derive(Clone)]
+#[derive(Clone, PartialEq, Debug)]
 #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
 pub struct DOMRefCell<T> {
     value: RefCell<T>,
 }
 
+// FIXME: These two impls make promises that are not quite true,
```

Comment:
Can you open an issue about this, and point to it, so there's a central place for discussing this?